node/request.js

```diff
@@ -198,6 +198,8 @@ TChannelRequest.prototype.send = function send(arg1, arg2, arg3, callback) {
 TChannelRequest.prototype.resend = function resend() {
     var self = this;
 
+    if (self.channel.destroyed) return;
```

Comment:
Actually, we cannot remove it. The channel may be already destroyed due to the retry ....